inventory/any.cf

```diff
@@ -608,6 +608,7 @@ bundle agent cfe_autorun_inventory_aws_ec2_metadata_cache
         comment => "This promise is used for delaying execution of url_get, since locking does not work directly on vars type promises",
         classes => kept_successful_command,
         handle => "daily_dummy_job_suppress_inform_incapable",
+        contain => in_shell,
```

Comment:
instead of running paths.true in a shell, perhaps we set a local variable for true and use that instead.

vars:
  "_true_command" string => ifelse( "windows", '$(sys.winsysdir)\cmd.exe /q /c "(exit 0)"', $(paths.true) );

Reply:
Right. The real problem here is preventing url_get from firing every agent run.

Would it make more sense to change this back to a wget commands promise which we could limit with `if_elapsed_day`? That would bring us back to having to have a different command for Windows as we did before.

Reply:
If we brought it back to using external tool I think it would 1, be more expensive than multple url_get calls and 2, make us hunt for an equivlant command to run to get the data.

What do you think about that suggestion of faking true out just inside this bundle to run cmd.exe /q /c exit 0?

Reply:
This seems like a good idea. I'll try it! :)